server.js

```diff
@@ -393,6 +393,66 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// continuousphp integration
+camp.route(/^\/continuousphp\/([^\/]+)\/([^\/]+\/[^\/]+)(?:\/(.+))?\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var provider = match[1];
+  var userRepo = match[2];
+  var branch   = match[3];
+  var format   = match[4];
+
+  var options = {
+    method: 'GET',
+    uri: 'https://status.continuousphp.com/' + provider + '/' + userRepo + '/status-info',
+    headers: {
+      'Accept': 'application/json'
+    }
+  };
+
+  if (branch != null) {
+    options.uri += '?branch=' + branch;
+  }
+
+  var badgeData = getBadgeData('build', data);
+  request(options, function(err, res) {
+    if (err != null) {
+      console.error('continuousphp error: ' + err.stack);
```

Comment:
Were these `console.error` calls only for debugging, or do you want to keep them at runtime too?

Reply:
Hello @Daniel15, thanks for reviewing our PR! If that's ok I'd like to keep the error handling. If there is another design implementation for the error handling that you would prefer, don't hesitate to get back to me :-)